### ISSUE TITLE
Official Docker builds

### ISSUE THREAD

**Issue by TaaviE** [zonemaster/zonemaster#1283]
It seems to be fairly involved to build and update zonemaster, are there any plans on making all the components available as pre-built Docker containers?

**Comment by matsduf** [comment]
We have everything in the public repositories. Instructions are found in https://github.com/zonemaster/zonemaster/blob/master/docs/internal/maintenance/ReleaseProcess-create-docker-image.md

Docker files:
* https://github.com/zonemaster/zonemaster-ldns/blob/master/Dockerfile
* https://github.com/zonemaster/zonemaster-engine/blob/master/Dockerfile
* https://github.com/zonemaster/zonemaster-cli/blob/master/Dockerfile

Is that what you ask for?

**Comment by TaaviE** [comment]
I have used the instructions to build my own images, but I think many would appreciate if already built and usable images were available from either Docker or GitHub registry.

**Comment by matsduf** [comment]
The Docker image is available on Docker Hub. Isn't that enough?

**Comment by TaaviE** [comment]
@tgreenx Yes, ideally the entire stack is available. Currently I see only Zonemaster-CLI.

**Comment by matsduf** [comment]
@TaaviE, we have discussed to make Zonemaster-Backend and Zonemaster-GUI available too, but it is not on the direct roadmap yet. I think you are the first to ask for it. Generally, there are very few questions about Zonemaster on Docker.

**Comment by TaaviE** [comment]
@matsduf It's good to hear it's at on the roadmap.

It might be that given the technical nature of the project many just build their own images instead of asking. There are also a few public repositories that contain relevant files (`docker-compose.yml` and `Dockerfile`) one can build upon. That's not to say there's no wish to use the images, if they'd be available.
